src/components/login/LoginForm.tsx

```diff
@@ -30,10 +37,20 @@ const LoginForm: React.FC<LoginForm> = ({ newMember }) => {
       }),
     });
     if (res.status === 401) setPasswordWrong(true);
-    if (res.status === 200) setOpen(true);
+    if (res.status === 200) setOpenModal(true);
```

Comment:
This is a personal preference, but I dislike single-line conditions. They obscure the complexity, because when I'm skimming through this code, it's not obvious to me that there is a logical branch here. Usually the levels of indentation tell a story about how complex code is, but that doesn't happen here.

Code that looks like this:

```
xx (xxXXxX xXxx xXX) {
  xxx xXX xxXXX
}
```

...immediately tells me that there is some sort of conditional branching happening, even though I haven't read and understood  the code itself (in this case of course the code is nonsense).

When the code just looks like this:

```
xxX xXx XXXX xxxXx Xx xXXXX;
```

I have to read that code carefully to realize that there is branching happening, and what code is involved in that branch (which might just be a single line, or if this line of code condtionally `return`s, the branch includes the rest of the function).